.ci/build.sh

```diff
@@ -5,7 +5,22 @@ SCRIPT_DIR="$(dirname "$0")"
 set -exuo pipefail
 source "$SCRIPT_DIR/common"
 
-docker build -t $CI_IMAGE_URI --build-arg IMAGE_REPOSITORY=${IMAGE} --build-arg IMAGE_TAG=${COMMIT} "$SCRIPT_DIR/.."
+export IMAGE_DESTINATION="registry"
+if [ "${CI_IMAGE_PLATFORMS}" = "local" ]; then
```

Comment:
oh, you had already made the local changes?

Reply:
Yeah, I'm improving the publishing step right now :)
We will have the final change in a few minutes 😁